interface/reports/collections_report.php

```diff
@@ -582,13 +584,13 @@ function checkAll(checked) {
                             <?php echo xlt('Aging Columns') ?>:
                         </td>
                         <td>
-                           <input type='text' name='form_age_cols' class='form-control' size='2' value='<?php echo attr($form_age_cols); ?>' />
+                           <input type='text' name='form_age_cols' class='form-control' size='2' value='<?php echo attr($form_age_cols) ? $form_age_cols : "3" ; ?>' />
                         </td>
                         <td class='control-label'>
                             <?php echo xlt('Days/Col') ?>:
                         </td>
                         <td>
-                           <input type='text' name='form_age_inc' class='form-control' size='3' value='<?php echo attr($form_age_inc); ?>' />
+                           <input type='text' name='form_age_inc' class='form-control' size='3' value='<?php echo attr($form_age_inc) ? $form_age_inc : "30"; ?>' />
```

Comment:
Need to escape the outputted value and not the comparison:
```
<input type='text' name='form_age_inc' class='form-control' size='3' value='<?php echo ($form_age_inc) ? attr($form_age_inc) : "30"; ?>' />
```

Reply:
Done